training/training/core/dataset.py

```diff
@@ -98,3 +103,97 @@ def getCategoryList(self) -> list[str]:
         if self._category_list is None:
             raise Exception("Category list not available")
         return self._category_list
+
+
+class DefaultImageDatasets(Enum):
+    MNIST = "MNIST"
+    FASHION_MNIST = "FashionMNIST"
+    KMNIST = "KMNIST"
+    CIFAR10 = "CIFAR10"
```

Comment:
are there any other torch default image datasets we can build in. will be a good beginner task for a new developer

Reply:
Yes! Here are a list of them. Lmk which one you would like:

CIFAR100: Similar to CIFAR10 but with 100 classes. Each class contains 600 images, 500 for training and 100 for testing.

SVHN (Street View House Numbers): A real-world image dataset for developing machine learning and object recognition algorithms with minimal requirement on data preprocessing and formatting.

ImageNet: A large visual database designed for use in visual object recognition software research. Note that ImageNet is much larger and more complex than the others.

CelebA (Celebrity Faces Attributes): A large-scale face attributes dataset with more than 200,000 celebrity images, each with 40 attribute annotations.

COIL100 (Columbia Object Image Library 100): A dataset consisting of 7200 images of 100 different objects. Each object was placed on a turntable and photographed at 5-degree intervals.

Omniglot: A dataset for one-shot learning. It contains 1623 different handwritten characters from 50 different alphabets.

STL10: An image recognition dataset for developing unsupervised feature learning, deep learning, self-taught learning algorithms. It is inspired by the CIFAR-10 dataset but with some modifications.

EMNIST (Extended MNIST): An expansion of the original MNIST dataset to include handwritten letters.

Reply:
can you make a beginner github issue